README.md

```diff
@@ -30,7 +30,7 @@ Here's a default config with annotations.
 # set this to any reachable IP or 0.0.0.0 to listen on all interfaces.
 host: localhost
 # The port can be changed in case it conflicts with any other application.
-port: 9940
+port: 9225
```

Comment:
Actually, the default port was changed to `9940` [a year ago](https://github.com/sladkoff/minecraft-prometheus-exporter/pull/157) but I haven't built a new release with that yet 😬 

Can you revert the port changes? I'll release a new version in the upcoming days.